Core/Core/Planner/GetCalendarEvents.swift

```diff
@@ -20,21 +20,29 @@ import Foundation
 
 public class GetCalendarEvents: CollectionUseCase {
     public typealias Model = CalendarEvent
-    public var cacheKey: String? { "\(context.pathComponent)/calendar-events/\(type.rawValue)" }
-    public let context: Context
+    public var cacheKey: String? { "\(contexts.first?.pathComponent ?? "")/calendar-events/\(type.rawValue)" }
```

Comment:
- The cache key should include all contexts. Imagine creating a UseCase with a single context, then a second one to a number of contexts and the second one's API call won't be made because the UseCase will think that it's already available in cache (since the first context matches) so it will pull the response from the first API call.
- Also the `importantDates` property should be included here for the same reasons.